src/components/images/blue-info-icon.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="20" height="20" viewBox="0 0 20 20" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path d="M10 0C4.48 0 0 4.48 0 10C0 15.52 4.48 20 10 20C15.52 20 20 15.52 20 10C20 4.48 15.52 0 10 0ZM11 15H9V9H11V15ZM11 7H9V5H11V7Z" fill="#045198"/>
+</svg>
```

Comment:
This file is redundant?

Reply:
There are also other svg files that are redundant, I added mine there so that people can have easy access to these icons if they wish to modify the source code.

Reply:
Yeah i just copied what you guys did